src/sentry/deletions/models/scheduleddeletion.py

```diff
@@ -32,6 +32,18 @@ def default_date_schedule() -> datetime:
     return timezone.now() + timedelta(days=30)
 
 
+# When models are moved, scheduled deletions will fail using the old model's
+# fully qualified name and will never be deleted. This mapping maps the old
+# names to the new name.
+RELOCATED_MODELS: dict[Tuple[str, str], Tuple[str, str]] = {
```

Comment:
Two things I don't love about this approach:

- It's only temporary needed until we flush out the old scheduled deletions
- This is super undiscoverable. Though if we added some documentation for moving models around we could probably solve the discoverability.

Reply:
Does it make sense to declare this on the model instead? Then have something that runs on startup, iterates all models and adds things to this dict?

Reply:
Yeah that sounds pretty good to me also. Maybe as part of the meta?

Reply:
Actually the more I think about this the more I think we should just keep this separate. I think it actually will end up being _harder_ to discover this if it's a property you can set on a model.